db/seeds.rb

```diff
@@ -149,6 +149,7 @@
   Judge.create!(first_name: 'Elena', last_name: 'Kagan', region_id: ny_region.id)
 
   #ActivityType
+  hearing = ActivityType.create!(name: "Hearing", cap: 0, accompaniment_eligible: false)
   check_in = ActivityType.create!(name: "Check In", cap: 3, accompaniment_eligible: true)
```

Comment:
I totally made up the names for these ActivityTypes - if these aren't realistic, let me know what we can make them. The important thing is that we have at least one with `accompaniment_eligible: false` and the other with `true` because one shows up on an admin's calendar and the other doesn't. _I think_